specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2024-03-01-preview/TuningOptions.json

```diff
@@ -0,0 +1,456 @@
+            "$ref": "./examples/Tuning_ListTuningOptions.json"
+          }
+        },
+        "description": "Retrieve the list of available tuning options.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../types/common-types.json#/parameters/ServerNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/TuningOptionsListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ImpactRecord": {
+      "type": "object",
+      "properties": {
+        "dimensionName": {
+          "type": "string",
+          "description": "Dimension name"
+        },
+        "unit": {
+          "type": "string",
+          "description": "Dimension unit"
+        },
+        "queryId": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Optional property that can be used to store the QueryId if the metric is for a specific query."
+        },
+        "absoluteValue": {
+          "type": "number",
+          "format": "double",
+          "description": "Absolute value"
+        }
+      },
+      "description": "Stores property that features impact on some metric if this recommended action is applied."
+    },
+    "IndexRecommendationDetails": {
+      "type": "object",
+      "properties": {
+        "databaseName": {
+          "type": "string",
+          "description": "Database name."
+        },
+        "schema": {
+          "type": "string",
+          "description": "Schema name."
+        },
+        "table": {
+          "type": "string",
+          "description": "Table name."
+        },
+        "indexType": {
+          "type": "string",
+          "description": "Index type."
+        },
+        "indexName": {
+          "type": "string",
+          "description": "Index name."
+        },
+        "indexColumns": {
+          "type": "string",
+          "description": "Index columns."
+        },
+        "includedColumns": {
+          "type": "string",
+          "description": "Index included columns."
```

Comment:
shouldn't here be an array? or at least add in the description how are the values separated

Reply:
Updated description to include will be semi-colon separated